src/mongo_logger.ts

```diff
@@ -383,7 +413,9 @@ export function stringifyWithMaxLen(
   maxDocumentLength: number,
   options: EJSONOptions = {}
 ): string {
-  let strToTruncate: string;
+  // eslint-disable-next-line @typescript-eslint/no-inferrable-types
+  let strToTruncate: string = '';
```

Comment:
```suggestion
  let strToTruncate = '';
```
You don't need to disable the eslint here. There is just no need to use an explicit type annotation.

Reply:
Ah I see!